src/pages/Result/Result.css

```diff
@@ -0,0 +1,20 @@
+.results {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.result-navigation {
+  width: 50%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 0.5em;
+  cursor: pointer;
```

Comment:
just to make sure you are using em it will adjust its size based on its parent element.

Reply:
Yes. current element font-size